app/controllers/attendance_records_controller.rb

```diff
@@ -19,6 +19,17 @@ def create
     end
   end
 
+  def update
+    @attendance_record = AttendanceRecord.find(params[:id])
+    @attendance_record.sign_out
```

Comment:
params should look like `{:attendance_record => {:sign_out => true, :student_id => 123}}`. then you can say `@attendance_record.update(attendance_record_params)` and let the model do the work.